src/planwise/component/maps.clj

```diff
@@ -73,6 +73,8 @@
   [service region-id facilities]
   (->> facilities
     (filter :polygon-id)
+    (filter :population)
```

Comment:
A value of `0` will pass the filter (since it's truthy) but will lead to a division by zero error later.